server/plugin/webhook.go

```diff
@@ -1045,3 +1049,40 @@ func (p *Plugin) handlePullRequestReviewNotification(event *github.PullRequestRe
 	p.CreateBotDMPost(authorUserID, message, "custom_git_review")
 	p.sendRefreshEvent(authorUserID)
 }
+
+func (p *Plugin) postStarEvent(event *github.StarEvent) {
+	repo := event.GetRepo()
+
+	subs := p.GetSubscribedChannelsForRepository(repo)
+
+	if len(subs) == 0 {
+		return
```

Comment:
would it be a good idea to add a log here to help with debugging and qa ?

Reply:
Hi, its reused pattern common throughout the plugin.